crnlib/lzma_LzmaEnc.cpp

```diff
@@ -1910,13 +1910,13 @@ SRes LzmaEnc_MemPrepare(CLzmaEncHandle pp, const Byte* src, SizeT srcLen,
   return LzmaEnc_AllocAndInit(p, keepWindowSize, alloc, allocBig);
 }
 
-void LzmaEnc_Finish(CLzmaEncHandle pp) {
 #ifdef COMPRESS_MF_MT
+void LzmaEnc_Finish(CLzmaEncHandle pp) {

```

Comment:
Using `(void)x` seems nicer here instead of writing the function declaration twice